packages/web/utils/error.ts

```diff
@@ -1,8 +1,6 @@
-import * as Sentry from "@sentry/nextjs";
-
 export function captureError(e: any) {
   if (e instanceof Error) {
-    Sentry.captureException(e);
+    // Sentry.captureException(e);
```

Comment:
Q: will exceptions be captured some other way?

Reply:
Yes, we'll use Opentelemetry. I updated the one on the `server` package but missed this one. Good catch! My latest commit fixes this